.github/workflows/file_health_exclusions_nonutf8.txt

```diff
@@ -0,0 +1,85 @@
+./dist/mac/CMakeLists.txt
+./dist/mac/qBitTorrentDocument.icns
+./dist/mac/qbittorrent_mac.icns
+./dist/qt-translations/qtbase_ar.qm
```

Comment:
It would be better to use patterns here.

Reply:
Or even hardcode exclusions of some file types.

Reply:
@glassez 

I'm afraid that could complicate things significantly for now, or at least it would force me to think about a different implementation. I think that's a good suggestion for a future improvement though. Let's keep things simple for now?

Reply:
It seems unreasonable to me that you should list every file if all files of this type should be excluded. This looks bad even for the initial implementation.

Reply:
fixed.